packages/web-components/src/components/va-table/test/va-table.e2e.ts

```diff
@@ -121,7 +121,7 @@ describe('va-table', () => {
   });
 
   it('right aligns columns with numeric data', async () => {
-    const rightAlignClass = 'medium-screen:vads-u-text-align--right';
+    const rightAlignClass = 'text-align-right';
```

Comment:
I am replacing the original Formation styling class with one that is contained within the component itself. This will guard us for when Formation is eventually removed.

Reply:
Love the proactive change!